src/coreclr/jit/emit.cpp

```diff
@@ -2736,13 +2736,29 @@ void emitter::emitSplit(emitLocation*         startLoc,
                 reportCandidate = false;
             }
 
+            // Don't report a zero-size candidate. This will only occur in a stress mode with JitSplitFunctionSize
```

Comment:
Can we also add an assert in `Alloc()` for 

```c++
assert(this->ufiEmitLoc->GetIG()->igSize > 0);
```

Reply:
Which `Alloc` are you referring to?

Reply:
sorry, I meant `Allocate()` method.

https://github.com/dotnet/runtime/blob/d5f59506354aa3b94574bc79bda21673a1da9774/src/coreclr/jit/unwindarm.cpp#L1631

Reply:
We can't have that assert, or any assert about the size of a particular IG, because the IG _could_ be zero sized, as long as there are other IGs in the fragment that have code. That's what the `endOffset > startOffset` is asserting.

Reply:
Hhm, that's true. There could be other non-zero IGs in the fragment.